components/ItemComponents/Content/MainMetadata.js

```diff
@@ -47,14 +47,12 @@ class MainMetadata extends React.Component {
     items should not have multiple rights but showing them in case a proper uri is present
     */
 
-    if (item.rights && Array.isArray(item.rights)) {
-      item.rights.map((theRight, index) => {
-        return <RightsBadge url={theRight} key={index} />;
-      })
+    if (item.edmRights) {
+      return <RightsBadge url={item.edmRights} />
+    } else if (item.rights && Array.isArray(item.rights)) {
```

Comment:
I might be being paranoid, but what if it's an empty array?

Reply:
Good point! How does this look?

```    
    if (item.edmRights) {
      return <RightsBadge url={item.edmRights} />
    } else if (Array.isArray(item.rights) && item.rights.length) {
      return <RightsBadge url={item.rights[0]} />;
    } else if (item.rights) {
      return <RightsBadge url={item.rights} />
    }
```

Reply:
Nevermind!